addons/binding/org.openhab.binding.amazonechocontrol/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,739 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="amazonechocontrol" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0" xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<bridge-type id="account">
+		<label>Amazon Account</label>
+		<description>Amazon Account where your amazon echo is registered.</description>
+		<config-description>
+			<parameter name="amazonSite" type="text" required="true">
+				<default>
+				</default>
+				<label>
+					Amazon site
+				</label>
+				<limitToOptions>
+					false
+				</limitToOptions>
+				<options>
+					<option value="amazon.de">
+						amazon.de
+					</option>
+					<option value="amazon.com">
+						amazon.com
+					</option>
+					<option value="amazon.ca">
+						amazon.ca
+					</option>
+					<option value="amazon.it">
+						amazon.it
+					</option>
+					<option value="amazon.fr">
+						amazon.fr
+					</option>
+					<option value="amazon.co.uk">
+						amazon.co.uk
+					</option>
+				</options>
+				<description>
+					Select the site where your amazon account is created.
+				</description>
+			</parameter>
+			<parameter name="email" type="text" required="true">
+				<default>
+				</default>
+				<label>
+					Amazon account email
+				</label>
+				<description>
+					Enter the email address of the amazon account which is used for the amazon echo devices. Hint: 2 factor authentication is not supported!
+				</description>
+			</parameter>
+			<parameter name="password" type="text" required="true">
+				<default>
+				</default>
+				<context>
+					password
+				</context>
+				<label>
+					Amazon account password
+				</label>
+				<description>
+					Enter the password of the amazon account which is used for the amazon echo devices. IMPORTANT: If the account thing does not go online and reports and login error, open the url YOUR_OPENHAP/amazonechocontrol/ID_OF_THIS_THING in your browser (e.g. http://openhab:8080/amazonechocontrol/account) and try to login.
+				</description>
+			</parameter>
+			<parameter name="pollingIntervalInSeconds" type="integer" min="10" max="65535" required="true" unit="s">
+				<default>
+					60
+				</default>
+				<label>
+					Refresh state interval
+				</label>
+				<description>
+					Refresh state interval in seconds. Lower time causes more network traffic.
+				</description>
+				<unitLabel>
+					Seconds
+				</unitLabel>
+			</parameter>
+			<!--

```

Comment:
Can you drop this commented out code

Reply:
I do not know whats the common practice for this. In this case, it is commented out, because this special feature should not be used from the public, because it seems depending on the used hardware drive (alexa drive) it does not work yet. But unfortunately, I do not own such I hardware yet, so I have to wait for a other developer which can finish debugging this. And so I thing, it would be the easiest way to remove it from the user, but have a way to simple reactivate it for testing. But If this is not a common practice, I could also remove this comment and all relevant classes (the smarthome handlers). Please let me know how I should continue.